galaxy_ng/app/api/ui/viewsets/execution_environment.py

```diff
@@ -212,9 +212,31 @@ def get_object(self):
         return self.queryset.get_or_create(container=distro)[0]
 
 
+class ContainerRegistryRemoteFilter(filterset.FilterSet):
+    name = filters.CharFilter(field_name='name')
+    url = filters.CharFilter(field_name='url')
+    name_icontains = filters.CharFilter(field_name='name', lookup_expr='icontains')
+    url_icontains = filters.CharFilter(field_name='url', lookup_expr='icontains')
```

Comment:
Here I think you don't need to specify the lookup expr, just the field.

```suggestion
    name = filters.CharFilter(field_name='name')
    url = filters.CharFilter(field_name='url')
```

Because on the `class Meta:` there is already a list of supported lookup expressions.

On api call that will enable `?name__icontains=foo` and `?name=foobar` and `?name__startswith=fo` 

I think @newswangerd knows more about filtering, but I am guessing this is how it works.